lib/src/main/java/space/taran/arklib/domain/index/ResourcesIndexRepo.kt

```diff
@@ -30,28 +25,31 @@ class ResourcesIndexRepo(
     private val provideMutex = Mutex()
     private val indexByRoot = mutableMapOf<Path, PlainResourcesIndex>()
 
-    private suspend fun loadFromDatabase(
+    private suspend fun create(
```

Comment:
Probably `load` or `provide` would be better. `Create` should be used if we construct values from scratch without usage of cached values. Am I right, this method either creates from scratch OR load from `.ark/index`?

Reply:
Yes, this method loads from `.ark/index`
Renamed to `load`